web-ui/src/components/dialogs/ConfirmationDialog.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
```

Comment:
Several components implemented their own confirmation dialog.
I created this reusable component and changed some components to use this instead of their own.